src/components/mgt-tasks/mgt-tasks.ts

```diff
@@ -1255,4 +1286,14 @@ export class MgtTasks extends MgtTemplatedComponent {
 
     return null;
   }
+
+  private calculateMediaQuery() {
```

Comment:
should this be a helper function?

Reply:
It will be. But let's put it in here for now.